config.mk

```diff
@@ -1,4 +1,4 @@
-PREFIX = /usr
+PREFIX = /usr/local
```

Comment:
`/usr` as a prefix is completely fine (and is standard)

Users who want `/usr/local` should invoke make with `make PREFIX=/usr/local install`.

I'm also against using `/usr/local` as the default prefix.